client/www/components/DocsPage.jsx

```diff
@@ -20,12 +20,19 @@ function collectHeadings(nodes, slugify = slugifyWithCounter()) {
   let sections = [];
 
   for (let node of nodes) {
-    if (node.name === 'h2' || node.name === 'h3') {
+    const isH2 =
+      node.name === 'h2' ||
+      (node.name === 'Heading' && node.attributes?.level === 2);
+    const isH3 =
+      node.name === 'h3' ||
+      (node.name === 'Heading' && node.attributes?.level === 3);
+
+    if (isH2 || isH3) {
       let title = getNodeText(node);
       if (title) {
         let id = slugify(title);
```

Comment:
Curious: it looks like we have `slugify` here. Should we use the same thing for `transformContentToId`?